discovery.md

```diff
@@ -156,7 +145,7 @@ Service:
   "protocols": [ "[string]" + ],
   "types": [ ?
     { *
-      "type": "[ce-type value]",
+      "name": "[ce-type value]",
```

Comment:
I wonder, if it is the attribute `type` that needs to be renamed to `name`, or if the name `types` of the array is misleading. Is it really a list of types? We are specifying how context attributes (including `type`) are used. So isn't it a list of event pattern or definitions?

Reply:
One more thing:
Would `name` be unique within the elements of `types`, or could there be multiple entries for the same event type?

Reply:
hmmm s/types/events/ might sounds interesting.  If we did that, would we keep "type"?  Seems like it might be ok then, no?

Reply:
Yeah events and type seems fine to me.